weaviate/values.yaml

```diff
@@ -2046,3 +2043,71 @@ affinity:
 ##
 priorityClassName: ""
 globalPriorityClassName: ""
+
+# Optionally enable a pod disruption budget for a given service.
+# One should define either minAvailable or maxUnavailable, but not both, as they are mutually exclusive.
+# Please note that pod disruption budgets are only applied when the corresponding module is enabled. 
+# For instance, if the text2vec-transformers module is disabled, its associated pod disruption budget will not be deployed.
+# A pod disruption budget is only activated when its respective module is enabled, allowing you to configure it as needed.
+# ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+podDisruptionBudgets:
+  weaviate:
+    enabled: false
+    spec:
+      minAvailable: 1
```

Comment:
Maybe we can set this minAvailable to 50% in the values.yaml to show taht it's also possible to configure percentages. Not a blocking thing though

Reply:
Thank you. I have set this value